service/directory-upload.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+usage="
+usage: $(basename "$0") command [-h] [-s] [-d] arguments...
+Uploads Measure and Library resources to the measure repository service.
+Options:
+    -h
+        Displays help menu.
+    -s [server baseUrl]
+        Specifies the base URL of the FHIR server to access.
+    -d
+        Provides directory path to parse for upload.
```

Comment:
I believe we can also use a file path
```suggestion
        Provides directory or file path to parse for upload.
```